tools/local_server/LocalServer.java

```diff
@@ -21,27 +23,30 @@
  * bazel-bin/tools/local_server/local_server -- {absolute_path}
  * {absolute_path} is absolute root path to serve files over (use `pwd` for current dir)
  */
+@Singleton
 public class LocalServer {
-  private static final Logger logger = Logger.getLogger(LocalServer.class.getName());
+  private static final FluentLogger logger = FluentLogger.forEnclosingClass();
 
   @FlagDesc(name = "local_server_port", description = "Port for local gRPC server")
   public static final Flag<Integer> localServerPort = Flag.create(8001);
 
   @FlagDesc(name = "root_path", description = "Root path for serving files for reviewer service")
-  public static final Flag<String> rootPath = Flag.create("");
+  public static final Flag<String> rootPath = Flag.create("/home/valerii/Work/storyteller/ok");
```

Comment:
Should probably revert `/home/valerii/Work/storyteller/ok` to "".

Reply:
Sorry. I forgot to remove this.